pkgs/tools/security/onionbalance/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub, buildPythonApplication
+, setuptools, setproctitle, stem, future, pyyaml, cryptography, pycrypto
+, pexpect, mock, pytest, pytest-mock, tox
+}:
+
+buildPythonApplication rec {
+  pname = "onionbalance";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "asn-d6";
+    repo = "onionbalance";
+    rev =  version;
+    sha256 = "0v2rrc7y6blxvsxg04yqn81sjwymi09wwm7mdgmh53zc0ryh9zv6";
+  };
+
+  buildInputs = [ setuptools setproctitle stem future pyyaml cryptography pycrypto ];
```

I got it to work with using `propagatedBuildInputs` instead of `buildInputs`

```suggestion
  buildInputs = [ setuptools ];
  propagatedBuildInputs = [ setproctitle stem pyyaml cryptography pycrypto future ];
```

---

It also needs setuptools in the propagatedBuildInputs. sorry for that